rules/Php80/Rector/Class_/AnnotationToAttributeRector.php

```diff
@@ -76,7 +76,7 @@ public function getRuleDefinition(): RuleDefinition
 class SymfonyRoute
 {
     /**
-     * @Route("/path", name="action")
+     * @Route("/path", name="action") api route
```

Comment:
Includes a sample of this functionality in the documentation

Reply:
I think separate fixture is better, and add next docblock example for `/** ` usage

Reply:
multi line comment fixture seems also needed

Reply:
Oh, i see, it is example config